HCore-Identity/Services/Impl/IdentityServicesImpl.cs

```diff
@@ -1614,6 +1735,35 @@ public async Task<UserModel> UpdateUserAsync(string userUuid, UserSpec userSpec,
 
                         await _signInManager.RefreshSignInAsync(oldUser).ConfigureAwait(false);
 
+                        if (_userNotificationProvider != null)
+                        {
+                            var userOldValues = new UserModel
+                            {
+                                FirstName = oldFirstName,
+                                LastName = oldLastName,
+                                PhoneNumber = oldPhoneNumber,
+                                NotificationCulture = oldNotificationCulture,
+                                GroupNotifications = oldGroupNotifications != null 
```

Comment:
Shouldnt "GroupNotifications" be null, if there has been no change?

Reply:
Yes, you are correct.
We can use the UserSpec class instead